xblock/runtime.py

```diff
@@ -886,6 +886,7 @@ def _wrap_ele(self, block, view, frag, extra_data=None):
         if frag.js_init_fn:
             data['init'] = frag.js_init_fn
```

Comment:
We need to think how to pass down the static file when this is being run in DEBUG mode, so that it can be served from the CDN. This will only be possible once this PR lands: https://github.com/edx/edx-platform/pull/10176

Another thing to note: XBlocks will serve uncached files in production with this change until we can work out how to use Optimizer. This is because RequireJS is unaware of the MD5 hashes and so will load files without them. These files are deliberately not cached in production because there is no way to know whether the content has changed. This is why we introduced Optimizer into the LMS in the first place -- see https://openedx.atlassian.net/wiki/display/TNL/RequireJS+in+the+LMS#RequireJSintheLMS-Background